packages/snap/src/index.ts

```diff
@@ -96,13 +96,19 @@ export const onRpcRequest: OnRpcRequestHandler = async ({ origin, request }) =>
             }
           })
         } catch (error) {
+
+          let message = error;
+          if (error.code === 'INSUFFICIENT_FUNDS') {
```

Comment:
You're not supposed to do this in typescript because the type of error is `any`, so there's no guarantee it has `.code`. See this article https://kentcdodds.com/blog/get-a-catch-block-error-message-with-typescript